templates/highly-efficient-record-processor.cfg.xml.erb

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<highly-efficient-record-processor xmlns="http://docs.openrepose.org/repose/highly-efficient-record-processor/v1.0"
+                                   pre-filter-logger-name="<%= @pre_filter_logger_name %>"
+                                   post-filter-logger-name="<%= @post_filter_logger_name %>"
+                                   service-code="<%= @service_code %>" region="<%= @region %>" data-center="<%= @datacenter %>">
+    <template crush="<%= @template_crush %>">
+<%- if @template.nil? -%>
+    <!-- todo: methodLabel, host.address -->
+    <![CDATA[
+        <?xml version="1.0"?>
+        <event xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
```

Comment:
Here's the full sample of a real atom entry:
https://github.com/rackerlabs/standard-usage-schemas/blob/bdb6e8271c4f9a185f95bfff689b61a2e41336ce/message_samples/corexsd/xml/identity-user-acess-event.xml

Remove line 2.